tests/src/test/java/alluxio/LocalAlluxioClusterResource.java

```diff
@@ -129,6 +131,8 @@ public LocalAlluxioClusterResource setProperty(PropertyKey key, Object value) {
   public void start() throws IOException, AlluxioException {
     // Init configuration for integration test
     mLocalAlluxioCluster.initConfiguration();
+    AuthenticatedClientUser.remove();
```

Comment:
logically the `initConfiguration()` call goes along with the calls to `Configuration.set`, so we should move these resets to the start of the method or to immediately before `start()`